index.js

```diff
@@ -144,20 +144,26 @@ class Replicate {
    * @returns {Promise<object>} - Resolves with the output of running the model
    */
   async run(ref, options, progress) {
+    let { block } = options;
     const { wait, signal, ...data } = options;
```

Comment:
You'll need to extract it on line 148 too otherwise it's going to be included in the `data` payload.

```suggestion
    let { block } = options;
    const { wait, signal, block: _, ...data } = options;
```

Reply:
Oh shoot. Yeah, good catch.